syntax = "proto2";

message s2u_player_heartbeat_response {
	required uint64 context = 1;
}